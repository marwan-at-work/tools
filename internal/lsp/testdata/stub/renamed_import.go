package stub

import (
	"context"
	mytime "time"
)

var _ context.Context = &myContext{} //@suggestedfix("&", "quickfix")
var _ = mytime.Time{}

type myContext struct{}
