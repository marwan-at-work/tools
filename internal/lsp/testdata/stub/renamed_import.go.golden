-- suggestedfix_renamed_import_8_25 --
package stub

import (
	"context"
	mytime "time"
)

var _ context.Context = &myContext{} //@suggestedfix("&", "quickfix")
var _ = mytime.Time{}

type myContext struct{}

// Deadline implements context.Context
func (*myContext) Deadline() (deadline mytime.Time, ok bool) {
	panic("unimplemented")
}

// Done implements context.Context
func (*myContext) Done() <-chan struct{} {
	panic("unimplemented")
}

// Err implements context.Context
func (*myContext) Err() error {
	panic("unimplemented")
}

// Value implements context.Context
func (*myContext) Value(key interface{}) interface{} {
	panic("unimplemented")
}

